service/orchestrator/metrics.json

```diff
@@ -316,6 +337,7 @@
     "id": "ObjectStoragePublicAccessDisabled",
     "name": "Object Storage: Public Access Disabled",
     "description": "This metric is used to assess if public acess is disabled for a object storage",
+    "category": "Identity Management",
```

Comment:
I think that is not really Identity Management, but i have not better idea at the moment. I think at the moment it is ok.